bucket/bucket_plain.go

```diff
@@ -5,7 +5,41 @@ import (
 )
 
 // MetricOperation is a list of metric operations to use for metric
-type MetricOperation [3]string
+type MetricOperation struct {
+	operations []string
```

Comment:
it used to have fixed size and now it is free-sized - how is it handled now?

Reply:
true thing, we are not checking it right now. Will push changes to handle that right now